docs/blog/microbit/arcadeshield-ext-sim.md

```diff
@@ -0,0 +1,37 @@
+# Arcade shield extension/simulator for the micro:bit (V2)
```

Comment:
I think the Foundation has stopped specifying "V2".  You could drop it from the title and just say in the first line, "the latest micro:bit.."
Also, (this is just a suggestion) but personally I would de-emphasize "Arcade" in the name of this extension.  It is just a very cool display extension for the micro:bit - doesn't really have to do with Arcade games.  I could see a lot of people using this without hardware either.

Reply:
Will remove V2. Of two minds about the "arcadeshield" vs "displayshield";  the shields are branded as "arcade"; understand for micro:bit users, it doesn't really matter and the simulator is good fun.  I will consider - renaming requires a little bit of re-programming.